source/content/files.md

```diff
@@ -4,15 +4,18 @@ description: Detailed information on how to access and optimize the Pantheon fil
 tags: [infrastructure, sftpfiles]
 categories: [platform,develop]
 ---
-Files are user uploads, usually images or documents. They are excluded from version control via Pantheon's .gitignore files <Popover content="The [.gitignore file](https://git-scm.com/docs/gitignore) is located at the root of the site's codebase and instructs Git which paths should be ignored." />:
+
+Files are large pieces of static content not stored in your database, usually images or documents, often user uploads. Because they are distinct from your site's [code](/code), they are excluded from version control via Pantheon's .gitignore files <Popover content="The [.gitignore file](https://git-scm.com/docs/gitignore) is located at the root of the site's codebase and instructs Git which paths should be ignored." />:
 
 - [Drupal 8](https://github.com/pantheon-systems/drops-8/blob/master/.gitignore)
 - [Drupal 7](https://github.com/pantheon-systems/drops-7/blob/master/.gitignore)
 - [WordPress](https://github.com/pantheon-systems/WordPress/blob/master/.gitignore)
 
 The Pantheon architecture comprises highly available [application containers](/application-containers) that are seamlessly integrated with Valhalla, our cloud-based filesystem. This means that your files are not local to the application containers running your site's codebase.
 
-Valhalla symbolically links the `wp-content/uploads` directory for WordPress and the `sites/default/files` directory for Drupal to the Valhalla files directory mount point, `/files` if viewed via an SFTP client. It is important to note that this directory is not part of the document root and is not directly web-accessible. Should you wish to make this accessible, you will need to create an additional symbolic link from within the document root.  Any [non-standard file locations](/non-standard-file-paths) should be symbolically linked to `/files` or moved manually.
+Valhalla creates a symbolic link (**symlink**), to the `files` directory at the appropriate location in your docroot (`wp-content/uploads` for WordPress and `sites/default/files` for Drupal).
+
+It is important to note that this directory is not part of the document root and is not directly web-accessible. If you need to make a path in `files` accessible from the docroot, you need to [create an additional symbolic link](/symlinks-assumed-write-access#create-a-symbolic-link) from within the document root.  Any non-standard file locations should be symbolically linked to `/files` or moved manually.
```

Comment:
This part is a bit confusing. It is true that `BDIR/files` is not web-accessible, but everything in the symlinked location (`wp-content/uploads` or `sites/default/files` or `files`) is.

The exception, of course, is the `private` directory inside of whichever location the files directory is symlinked at, which is explicitly blocked. However, if the user makes their own symlink to the `files` directory, then the private directory in the new location they opened up *is* going to be web accessible. There are instructions on how the user can block directives in the pantheon.yml docs.

Reply:
Sounds like the solution here is to change "this directory" to "The files directory in your site root", to make the distinction between what is and isn't being defined as in accessible from the web.

Reply:
Adjusted as discussed in https://github.com/pantheon-systems/documentation/pull/5575/commits/39603a2136bc0d5b79bdcc7aae826037e344391e